src/Record/Unsafe.ss

```diff
@@ -5,33 +5,32 @@
           unsafeGet
           unsafeSet
           unsafeDelete)
-  (import (only (rnrs base) define lambda let)
-          (prefix (purs runtime) rt:)
-          (prefix (purs runtime srfi :125) srfi:125:))
+  (import (chezscheme)
+          (prefix (purs runtime) rt:))
 
   (define unsafeHas
     (lambda (label)
       (lambda (rec)
-        (srfi:125:hash-table-contains? rec label))))
+        (symbol-hashtable-contains? rec (string->symbol label)))))
```

Comment:
I guess we're not going for `bytestring->symbol` just yet? Or will this not change?

Reply:
This is just for being compatible with new symbol-based hashtables, we don't have bytestrings yet.